File: 프로그래머스/unrated/120811. 중앙값 구하기/중앙값 구하기.py
def solution(array):
    array.sort()  # 리스트를 오름차순으로 정렬
    return array[len(array)//2]  # 중앙값 반환

print(solution([1, 2, 7, 10, 11]))  # 출력: 7

File: 프로그래머스/Lv.0/120811. 중앙값 구하기/중앙값 구하기.java
import java.util.Arrays;

public class Solution {
    public int solution(int[] array) {
        Arrays.sort(array);
        int length = array.length;
        if (length % 2 == 0) {
            return (array[length/2 - 1] + array[length/2]) / 2;
        } else {
            return array[length/2];
        }
    }
}
